ddp-workspace/projects/ddp-singular/playwright-e2e/README.md

```diff
@@ -0,0 +1,99 @@
+# DDP Singular project test automation with Playwright
+
+Playwright end-to-end (E2E) UI integration tests.
+
+
+* See [Playwright Test Runner](https://playwright.dev/docs/api/class-test)
+* See [Playwright API](https://playwright.dev/docs/api/class-playwright)
+
+
+## Set up *playwright-e2e* project
+
+* Playwright [Node.js Requirement](https://playwright.dev/docs/troubleshooting#nodejs-requirements)
+* Install `node.js` and `yarn` if not already installed 
+  * Install the latest version of `Node.js`. See [Node.js Install](https://nodejs.dev/en/learn/how-to-install-nodejs/)
+  * Install the latest version of `yarn`. See [Yarn Install](https://yarnpkg.com/getting-started/install)
+* Clone github ddp-angular project: 
+  * `git clone git@github.com:broadinstitute/ddp-angular.git`
+* Install dependencies:
+  * `cd ddp-angular/ddp-workspace/projects/ddp-singular/playwright-e2e`
+  * `yarn install`
```

Comment:
Wondering can we use `npm` instead of `yarn` ?
just because we use npm everywhere else.

Reply:
Good point. I don't have any issue with switching. `yarn` is used by terra-ui, all-of-us, etc. I don't know any terra team that uses `npm`. Let's talk about this in standup.

Reply:
Discussed in Standup. Switched from `yarn` to `npm`.